app/helpers/react_components/track/credits.rb

```diff
@@ -0,0 +1,19 @@
+module ReactComponents
+  module Track
+    class Credits < ReactComponent
+      initialize_with :avatar_urls, :top_count, :top_label, :bottom_count, :bottom_label, max: 2
+
+      def to_s
+        super("track-credits", {
+          avatar_urls:,
+          top_count:,
+          top_label:,
+          bottom_count:,
+          bottom_label:,
+          max:
+        }
```

Comment:
You probably don't need the `{` and `}`, but if you do, then this one and the line below should be on the same line (to match how they are on L7)